### PR TITLE
Backend development

### PR DESCRIPTION
**Pull request by CyberDragonWizard**

### CODE REVIEW

**Review by mcorrieri — APPROVED** [review]
Mongo db backend?

**Review by mikenester — CHANGES_REQUESTED** [review]
Add a .gitignore file and ignore the `node_modules `directory and the `.env` file. `.env` should never be in source control because it will usually contain secrets, like API Keys.
https://www.atlassian.com/git/tutorials/saving-changes/gitignore

File: backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.env
+
+node_modules/
```
Comment (mikenester) [inline backend/.gitignore:3]: this didn't work

Comment (mikenester) [reply backend/.gitignore:3]: delete this file

Comment (CyberDragonWizard) [reply backend/.gitignore:3]: Ill try again

File: backend/routes/Post.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const User = '../models/User';
+const {Post} = require('../models/Post');
+
+
+// get all 'Tweets'
+router.get('/', async (req, res) => {
+    const posts = await Post.find().sort({ date: -1});
+    
+    if (!posts) {
+        res.status(500).json({success: false})
```
Comment (mikenester) [inline backend/routes/Post.js:12]: 500 is an internal server error. this should be 400. You never want to intentionally throw a 500. Is the app crashes, it auto throws 500

**Review by mikenester — APPROVED** [review]

**Pull request merged**